ext/include/opentelemetry/ext/http/common/url_parser.h

```diff
@@ -72,15 +73,19 @@ class UrlParser
     {
       // port missing. Used default 80 / 443
       if (scheme_ == "http")
+      {
         port_ = 80;
-      if (scheme_ == "https")
+      }
+      else if (scheme_ == "https")
```

Comment:
Added `else` here because otherwise, we will have two comparisons for `http`, and the second one will be unnecessary because we already have the value.